setup.py

```diff
@@ -11,7 +11,12 @@
     description="Workflow orchestration and management.",
     author="Prefect Technologies, Inc.",
     author_email="help@prefect.io",
-    url="https://github.com/PrefectHQ/prefect/",
+    url="https://www.prefect.io",
+    project_urls={
+        "Documentation": "https://docs.prefect.io",
+        "Source": "https://github.com/PrefectHQ/prefect",
```

Comment:
Should we just call this one "GitHub" or is "Source" more common?

Reply:
Seems source is more used: https://github.com/pypi/warehouse/issues/5947#issuecomment-699660629

But i can switch to Github if you want. We also have the `Changelog` that render like this on PyPI:

![image](https://user-images.githubusercontent.com/5853172/214125362-3506d2e9-7402-4e4d-94ec-2039ca8c2045.png)

Reply:
I just need the  Changelog url, if we want to add too.

Reply:
"Source" seems fine, thanks for the reference!

There's a changelog at https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md